hackathon_site/hackathon_site/tests.py

```diff
@@ -62,74 +76,67 @@ def _get_random_email():
+        return [user1, user2, user3, user4]
+
+    def _make_full_registration_team(self, team=None, self_users=True, num_users=4):
         if team is None:
             team = RegistrationTeam.objects.create()
-
-        if self_users:
-            user1 = self.user
-            user2 = self.user2 = User.objects.create_user(
-                username="frank@johnston.com",
-                password="hellothere31415",
-                email="frank@johnston.com",
-                first_name="Frank",
-                last_name="Johnston",
-            )
-            user3 = self.user3 = User.objects.create_user(
-                username="franklin@carmichael.com",
-                password="supersecret456",
-                email="franklin@carmichael.com",
-                first_name="Franklin",
-                last_name="Carmichael",
-            )
-            user4 = self.user4 = User.objects.create_user(
-                username="lawren@harris.com",
-                password="wxyz7890",
-                email="lawren@harris.com",
-                first_name="Lawren",
-                last_name="Harris",
-            )
+        if self_users == True:
+            for user in self.create_user_set():
+                self._apply_as_user(user, team)
         else:
-            # Make some random users
-            email1 = self._get_random_email()
-            email2 = self._get_random_email()
-            email3 = self._get_random_email()
-            email4 = self._get_random_email()
-
-            user1 = User.objects.create_user(
-                username=email1,
-                password="foobar123",
-                email=email1,
-                first_name="John1",
-                last_name="Doe1",
-            )
-            user2 = User.objects.create_user(
-                username=email2,
-                password="foobar123",
-                email=email2,
-                first_name="John2",
-                last_name="Doe2",
-            )
-            user3 = User.objects.create_user(
-                username=email3,
-                password="foobar123",
-                email=email3,
-                first_name="John3",
-                last_name="Doe3",
-            )
-            user4 = User.objects.create_user(
-                username=email4,
-                password="foobar123",
-                email=email4,
-                first_name="John4",
-                last_name="Doe4",
-            )
-
-        self._apply_as_user(user1, team)
-        self._apply_as_user(user2, team)
-        self._apply_as_user(user3, team)
-        self._apply_as_user(user4, team)
+            for user_number in range(1, num_users + 1, 1):
+                random_email = self._get_random_email()
+                new_user = User.objects.create_user(
+                    username=random_email,
+                    password="foobar123",
+                    email=random_email,
+                    first_name="John{}".format(user_number),
```

Comment:
Slightly more condensed:

```suggestion
                    first_name=f"John{user_number}",
```

Same with other format strings below